src/current/v24.3/ui-storage-dashboard.md

```diff
@@ -77,24 +89,46 @@ If you are running multiple nodes on a single machine (not recommended), the act
 
 Graph | Description
 --------|--------
-Log Commit Latency: 99th Percentile | The 99th percentile latency for commits to the Raft log.
-Log Commit Latency: 50th Percentile | The 50th percentile latency for commits to the Raft log.
-Command Commit Latency: 99th Percentile | The 99th percentile latency for commits of Raft commands.
-Command Commit Latency: 50th Percentile | The 50th percentile latency for commits of Raft commands.
+WAL Fsync Latency | The latency for fsyncs to the storage engine's write-ahead log.
+Log Commit Latency: 99th Percentile | The 99th percentile latency for commits to the Raft log. This measures essentially an fdatasync to the storage engine's write-ahead log.
+Log Commit Latency: 50th Percentile | The 50th percentile latency for commits to the Raft log. This measures essentially an fdatasync to the storage engine's write-ahead log.
+Command Commit Latency: 99th Percentile | The 99th percentile latency for commits of Raft commands. This measures applying a batch to the storage engine (including writes to the write-ahead log), but no fsync.
+Command Commit Latency: 50th Percentile | The 50th percentile latency for commits of Raft commands.  This measures applying a batch to the storage engine (including writes to the write-ahead log), but no fsync.
 Read Amplification | The average number of real read operations executed per logical read operation across all nodes. See [Read Amplification]({% link {{ page.version.version }}/architecture/storage-layer.md %}#read-amplification).
 SSTables | The number of SSTables in use across all nodes.
 Flushes | Bytes written by [memtable flushes]({% link {{ page.version.version }}/architecture/storage-layer.md %}#memtable-and-write-ahead-log) across all nodes.
+WAL Bytes Written | Bytes written to WAL files across all nodes.
 Compactions | Bytes written by [compactions]({% link {{ page.version.version }}/architecture/storage-layer.md %}#compaction) across all nodes.
-Ingestions | Bytes written by SSTable injections across all nodes.
+Ingestions | Bytes written by SSTable ingestions across all nodes.
 Write Stalls | The number of intentional write stalls per second across all nodes, used to backpressure incoming writes during periods of heavy write traffic.
 Time Series Writes | The number of successfully written time-series samples, and number of errors attempting to write time series samples, per second across all nodes.
-Time Series Bytes Written | The number of bytes written by the time-series system per second across all nodes.
+Time Series Bytes Written | The number of bytes written by the time-series system per second across all nodes.<br><br>Note that this does not reflect the rate at which disk space is consumed by time series; the data is highly compressed on disk. This rate is instead intended to indicate the amount of network traffic and disk activity generated by time series writes.
```

Comment:
when "time series" is modifying a noun, should it be "time-series"? There is "time-series system" and "time series writes" here — not sure which one you want to go for?